From 462f030d52e6245d89e9be98bce2f0fae97efc67 Mon Sep 17 00:00:00 2001
From: Xiaowei Bao <xiaowei.bao@nxp.com>
Date: Tue, 4 Dec 2018 10:35:42 +0800
Subject: [PATCH 419/741] arm64: dts: ls1046a: Remove the IB and OB property

commit 46999a9d9d93507dd426ad42e70bd875b93c8348 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Remove the IB and OB property from the PCIe RC node, these two property
will add to the PCIe EP node later, These two properties are for internal
testing and opensource is deprecated.

Signed-off-by: Xiaowei Bao <xiaowei.bao@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
index 3ae1621f1d9c..64796cb1905e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
@@ -708,8 +708,6 @@
 			iommu-map = <0 &smmu 0 1>;
 			num-lanes = <4>;
 			num-viewport = <8>;
-			num-ib-windows = <6>;
-			num-ob-windows = <6>;
 			bus-range = <0x0 0xff>;
 			ranges = <0x81000000 0x0 0x00000000 0x40 0x00010000 0x0 0x00010000   /* downstream I/O */
 				  0x82000000 0x0 0x40000000 0x40 0x40000000 0x0 0x40000000>; /* non-prefetchable memory */
@@ -749,8 +747,6 @@
 			iommu-map = <0 &smmu 0 1>;
 			num-lanes = <2>;
 			num-viewport = <8>;
-                        num-ib-windows = <6>;
-                        num-ob-windows = <6>;
 			bus-range = <0x0 0xff>;
 			ranges = <0x81000000 0x0 0x00000000 0x48 0x00010000 0x0 0x00010000   /* downstream I/O */
 				  0x82000000 0x0 0x40000000 0x48 0x40000000 0x0 0x40000000>; /* non-prefetchable memory */
@@ -790,8 +786,6 @@
 			iommu-map = <0 &smmu 0 1>;
 			num-lanes = <2>;
 			num-viewport = <8>;
-                        num-ib-windows = <6>;
-                        num-ob-windows = <6>;
 			bus-range = <0x0 0xff>;
 			ranges = <0x81000000 0x0 0x00000000 0x50 0x00010000 0x0 0x00010000   /* downstream I/O */
 				  0x82000000 0x0 0x40000000 0x50 0x40000000 0x0 0x40000000>; /* non-prefetchable memory */
-- 
2.17.1

